src/components/DisplayInput.js

```diff
@@ -128,28 +133,42 @@ const DisplayInput = React.createClass({
         }
       },
 
-      label: {
-        ':focus': {
-          color: this.props.primaryColor
-        }
+      inputWrapper: {
+        alignItems: 'center',
+        display: 'flex',
+        paddingBottom: StyleConstants.Spacing.SMALL,
```

Comment:
The only concern I had with the short hand is if we have to override one. That may not be an issue, though or we can use the short hand to do that. It just makes the use a little more coupled because we have to know all the padding and make sure we pass in the same amounts for the ones we don't want to override.